src/utils/constants.ts

```diff
@@ -1,5 +1,8 @@
 export const WOC_BASE_URL = "https://api.whatsonchain.com/v1/bsv/main";
-export const GP_BASE_URL = "https://ordinals.gorillapool.io/api";
+export const GP_BASE_URL = "https://v3.ordinals.gorillapool.io/api";
```

Comment:
Would prefer to keep just one `GP_BASE_URL` and just append `/api...` or `/content` instead of also needing a CONTENT_BASE_URL as well.